onnxruntime/core/graph/function.cc

```diff
@@ -1,14 +1,15 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
+#include "core/framework/inlined_containers.h"
```

Comment:
could inlined_containers.h move to somewhere like "core/common/inlined_containers.h"?
onnxruntime_graph shouldn't depend on onnxruntime_framework (and it doesn't here since that header doesn't have implementation in onnxruntime_framework, but it looks like it does)
https://github.com/microsoft/onnxruntime/blob/bfb20b315d9763f4113fdb1b15444626d66874e6/cmake/onnxruntime.cmake#L160-L186

Reply:
Feels right to me